src/components/common/heading/heading.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Our common components should be located in `components` folder. Screens will be in `screens` folder.